test/unit/services/sailsResponseProcessor_spec.js

```diff
@@ -10,7 +10,12 @@ describe('sailsResponseProcessor', function () {
         id: '1',
         attributes: {
           capacity: 10, availability: 4,
-          expectedins: [{id: 1}, {id: 2}]
+          expectedins: [{id: 1}, {id: 2}],
+          PrebookingDetail: {
```

Comment:
It can probably be a lower case `p` (and also `d`), but this was originally done this way to give the `$apply` something to denote special cases for merging.